spec/protocol/endpoints.adoc

```diff
@@ -0,0 +1,19 @@
+= MANY Endpoints
+
+A server exposes endpoints that a message must target.
```

Comment:
Endpoints are methods exposed by servers that a client can trigger to perform an operation.
A client can trigger an endpoint by sending the server a correctly-formatted `bytestring` payload. (Add a link to the spec/CDDL)
The server will reply a `bytestring` to the client after the operation completes. The reply may contain the result of the operation or an error if the operation failed to execute properly.
Endpoints follow the `module.endpoint` naming convention, e.g., `ledger.send`.


Or something like that.

Reply:
>  Endpoints follow the module.endpoint naming convention, e.g., ledger.send.

It should, but it doesn't have to, especially for endpoints that don't care about the spec attributes. If you're making endpoints for a private server or something, you don't need to do that.

Reply:
I'm in the process of rewording, I think, but we can also improve on wording later on.